{
  "tests": [
    {
      "query_file": "success.graphql",
      "result": {
        "data": { "restartTask": { "status": "will-run", "id": "task1" } }
      }
    },
    {
      "query_file": "error-invalid-task_id.graphql",
      "result": {
        "errors": [
          {
            "message": "error restarting task nonexistent: cannot restart task nonexistent because it could not be found",
            "path": ["restartTask"],
            "extensions": { "code": "INTERNAL_SERVER_ERROR" }
          }
        ],
        "data": null
      }
    }
  ]
}
